### PR TITLE
Doc update for setting an input parameter as the return value

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/parameter.md [hunk 1]
```diff
@@ -10,18 +10,28 @@ tags: ["studio pro", "parameter", "microflow"]
...
-### 2.2 Data Type
+### 3.1 Data Type
 
 The data type of a parameter defines the type of the value that it expects. See [Data Types](/refguide/data-types/) for the possible data types.
 
 Default: *Object*
+
+### 3.2 Name
+
+**Name** refers to the value of the parameter.
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/parameter.md]: Shouldn't it be `refers to the name of the value of the parameter` or `the name of the parameter`?

Comment (ghost) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/parameter.md]: It should indeed be the name of the parameter

**Review by ghost — COMMENTED** [review]
Reference around name is wrong. everything else seems to be correct.

**Comment by Yiyun333** [comment]
> Reference around name is wrong. everything else seems to be correct.

Thanks for the review! The description of the name is corrected now :)

**Pull request merged**